code/api/src/modules/user/resolvers.js

```diff
@@ -77,3 +77,42 @@ export async function remove(parentValue, { id }) {
 export async function getGenders() {
   return Object.values(params.user.gender)
 }
+
+// Update User styleSummary
+export async function updateStyleSummary(parentValue, { id, styleSurvey }) {
+  // tally number of occurrences for each style into a new hash
+  const formSummary = styleSurvey.reduce(function (acc, style) {
+    if (typeof acc[style] == 'undefined') {
+      acc[style] = 1;
```

Comment:
This resolver looks like it could be cleaned up. If we have time we should go through this with Trond and Quinn, their JS experience would be very useful for this.